docs/segment.md

```diff
@@ -2,98 +2,264 @@
+
+- Cutting through one or several text lines using the scissors tool (++c++). This is particularly helpful for documents with multiple columns. But, you will only be able to draw a straight rectangle so if your image is skewed, it will require more work. 
+
+![image: Cutting a text line.](img/segment/scissors.gif "You can cut one or several text lines using the scissors tool (++c++).")
+
+- Inverting the reading direction of a line by selecting it, then clicking on the **reverse selected lines** button or by pressing ++i++ (not available for regions).
+
+- Merging two or more active lines by clicking on the **join selected lines** button or pressing ++j++ (not available for regions).
+
+!!! tip
+    
+    All advanced features are described in a help window, just click on **`?`** in the Segmentation panel. 
+
+    You should also check the shortcut page in this documentation! <!-- todo: add link to shortcut page -->
+
+
+### Draw the masks
+
+By default, only the baselines are shown in the "Segmentation" panel: you can toggle the display of the masks by clicking on the "mask button" (shortcut ++m++). Masks appear as more or less complex polygons, sometimes missing the highest or lowest part of the letters, or missing diacritics. 
+
+eScriptorium asynchronously calculates the mask associated with each segment. When it is the first time that a line is manually drawn on an image, you have to manually trigger the mask calculation. The reason is to let you finish drawing all the baselines you want before starting a background job. When mask calculation is not automatically active, a **thumbs up** button will appear at the top of the panel. Simply click on the **thumbs up** button to start it. 
+
+![image: Triggering the automatic calculation of masks](img/segment/calculate_masks.gif "You can manually activate the mask calculation by pressing the calculate masks button.")
+
+!!! warning "Don't fix the masks"
+    
+    Unless you have a good reason to, it is not recommanded to manually modify the masks: it would be a tedious task and the risk of the modifications being overwritten by an automatic mask calculation is too high.
+
+## Reordering lines
+
+As mentioned above, regions and lines are ordered. The reading order of the text lines is calculated automatically. There are two ways to control line order:
+
+- You can display the reading order of the lines on the “Segmentation” panel by clicking on the **Toggle ordering display** button or by pressing ++l++.
+
+- You can also use the “Text” panel (shortcut ++ctrl+5++) where the transcription will appear as a succession of lines. Each line is associated to a number, corresponding to the reading order.
+
+Reading order doesn't affect the transcription task but it will affect how the text is exported. In some cases correcting the reading order is not important but you might want to correct it. 
```

Comment:
For the second phrase, shouldn't it be "In most cases" rather than "In some cases" to make the rest of the sentence more logical ?

Reply:
Hm, I don't agree with this one because in many cases the reading order is not important, but it really depends on what the data is used for after.

Reply:
I think we are saying the same thing, since, as you say, in many cases the reading order is not important, the last sentence would make more sense with "most" rather than "some".
Maybe we can add a little more like "In most cases, correcting the reading order is not important, but if it is, you might want to correct it" or is it too much ?

Reply:
what about this: "Reading order doesn't affect the transcription task but it will affect how the text is exported. In some cases correcting the reading order is crucial so you will want to control and correct it, while in other cases it is not important or not necessary." ?

Reply:
no wait, that's repetitive